fe/fe-core/src/main/java/com/starrocks/plugin/AuditEvent.java

```diff
@@ -115,9 +115,9 @@ public enum EventType {
     @AuditField(value = "Digest")
     public String digest = "";
     @AuditField(value = "PlanCpuCost")
-    public double planCpuCosts = -1;
+    public long planCpuCosts = -1;
     @AuditField(value = "PlanMemCost")
-    public double planMemCosts = -1;
+    public long planMemCosts = -1;
```

Comment:
It is dangerous to change it to long it is very easy to overflow. If you just want it not human friendly you can only change the output format of double